spek-runtime/src/commonMain/kotlin/org/spekframework/spek2/launcher/ConsoleLauncher.kt

```diff
@@ -0,0 +1,119 @@
+package org.spekframework.spek2.launcher
+
+import org.spekframework.spek2.launcher.reporter.BasicConsoleReporter
+import org.spekframework.spek2.runtime.SpekRuntime
+import org.spekframework.spek2.runtime.discovery.DiscoveryContext
+import org.spekframework.spek2.runtime.execution.DiscoveryRequest
+import org.spekframework.spek2.runtime.execution.ExecutionListener
+import org.spekframework.spek2.runtime.execution.ExecutionRequest
+import org.spekframework.spek2.runtime.execution.ExecutionResult
+import org.spekframework.spek2.runtime.scope.GroupScopeImpl
+import org.spekframework.spek2.runtime.scope.Path
+import org.spekframework.spek2.runtime.scope.PathBuilder
+import org.spekframework.spek2.runtime.scope.TestScopeImpl
+
+sealed class ReporterType
+class ConsoleReporterType(val format: Format): ReporterType() {
+    enum class Format {
+        BASIC,
+        SERVICE_MESSAGE
+    }
+}
+
+data class LauncherArgs(
+    val reporterTypes: List<ReporterType>,
```

Comment:
In what scenario would you want to have multiple reporters?

Reply:
In gradle, sometimes you'd want to generate `xml` and `html` reports at the same time.

Reply:
Right, makes sense.